src/App.js

```diff
@@ -10,25 +10,30 @@ class App extends Component {
   constructor(props) {
     super(props)
     this.state = {
-      authorization: {}
+      authenticationValues: {}
```

Comment:
authenticationInfo ir authInfo sounds like a better name instead of 'values'.

Reply:
Done.